libbeat/asset/asset.go

```diff
@@ -65,3 +70,61 @@ type Data struct {
 	Data     string
 	Package  string
 }
+
+func (d *Data) SetName(name string) {
+	d.Name = goTypeName(name)
+}
+
+func CreateAsset(license string, beat string, name string, p string, data []byte, priority string) ([]byte, error) {
```

Comment:
what `package` rather than `p`? ;-)

Reply:
Got it. But still, especially for a method signature, you should find a representative name. Could be `pkg`, as I've seen you used elsewhere, when calling this function.

Reply:
Agree, I like `pkg`, changed it.